src/super_gradients/training/models/pose_estimation_models/dekr_hrnet.py

```diff
@@ -520,8 +530,102 @@ def init_weights(self):
                 if hasattr(m, "bias"):
                     nn.init.constant_(m.translation_conv.bias, 0)
 
+    @staticmethod
+    def get_post_prediction_callback(conf: float = 0.05):
+        return DEKRPoseEstimationDecodeCallback(
+            min_confidence=conf,
+            keypoint_threshold=0.05,
+            nms_threshold=0.05,
+            apply_sigmoid=True,
+            max_num_people=30,
+            nms_num_threshold=8,
+            output_stride=4,
+        )
+
+    @resolve_param("image_processor", ProcessingFactory())
+    def set_dataset_processing_params(
+        self,
+        joint_links: Union[np.ndarray, List[Tuple[int, int]]],
+        joint_colors: Union[np.ndarray, List[Tuple[int, int, int]]],
+        keypoint_colors: Union[np.ndarray, List[Tuple[int, int, int]]],
+        image_processor: Optional[Processing] = None,
+        conf: Optional[float] = None,
+    ) -> None:
+        """Set the processing parameters for the dataset.
+
+        :param class_names:     (Optional) Names of the dataset the model was trained on.
+        :param image_processor: (Optional) Image processing objects to reproduce the dataset preprocessing used for training.
+        :param iou:             (Optional) IoU threshold for the nms algorithm
+        :param conf:            (Optional) Below the confidence threshold, prediction are discarded
+        """
+        self._joint_links = joint_links or self._joint_links
+        self._joint_colors = joint_colors or self._joint_colors
+        self._keypoint_colors = keypoint_colors or self._keypoint_colors
+        self._image_processor = image_processor or self._image_processor
+        self._default_nms_conf = conf or self._default_nms_conf
+
+    @lru_cache(maxsize=1)
```

Comment:
what is this for?

Reply:
@lru_cache(maxsize=1) ?
Copied from detection pipeline. Idea is to cache the pipeline and not re-instantiate it at every call.